packages/itwin/reports-config-widget/src/widget/hooks/useActiveIModel.ts

```diff
@@ -0,0 +1,18 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import { useActiveIModelConnection } from "@itwin/appui-react";
+
+export interface ActiveIModel {
+  iTwinId: string;
+  iModelId: string;
+}
+// Wrapped in a hook for easy mocking
+export const useActiveIModel = () => {
```

Comment:
what is even the point of this hook? just to wrap and return the ids ina subset of IModelConnection?

Reply:
This was me giving up on trying to mock out this hook within iTwin.js for tests.